index.html

```diff
@@ -0,0 +1,267 @@
+    </div>
+    <div id="answer4" class="answer">
+      <p class="answer-p">
+        As a <span class="red">period sponsor</span> you contributes to ensure that free <span class="red">menstrual
+        hygiene</span> products are handed out.
+        By doing so, you make sure so that <span class="red">period poverty</span> does not affect
+        schooling or get in the way of physical and mental well-being.<br><br>
+        You will also make a real impact, break the taboo, and create awareness to <span class="red">period
+        poverty</span>!
+      </p>
+    </div>
+
+    <div class="question" id="section5">
+      <h3 class="accordion-h3">So, what's the impact?</h3>
+    </div>
+    <div id="answer5" class="answer">
+      <p class="answer-p">Remember those 10 000 hygine products and the 63 000 SEK?
+        Imagine that half of your FB friends became <span class="red">period sponsors</span>
+      </p>
+      <img class="accordion-image" src="./images/pp.png" alt="graph over proposed impact">
+    </div>
+  </section>
+
+
+  <!-- Signup form -->
+
+  <section class="the-form" id="the-form">
+
+    <fieldset class="field-container">
+      <legend>
+        <h2 class="manifesto-h2">choose your impact!</h2>
+      </legend>
+      <p class="p-form-main">
+        As a <span class="red">menstrual sponsor</span>, you ensure that a young person can buy <span
+          class="red">menstrual hygiene</span> products every month.
+        A package of pads or tampons costs approximately 28 SEK .
+        You decide how many packages you want and can contribute with. </p>
+
+      <form class=”form” action="http://httpbin.org/anything" method=”POST”>
+        <p class="p-form">SELECT AN AMOUNT</p>
+
+        <div class="amount-container">
+          <div class="amount">
+            <label for="SekValue" class="radio-button-container">
+              <p class="p-form-small">28 sek/month</p>
+              <input id="28SEK" type="radio" name="SekValue" checked>
+              <span class="custom-radio"></span>
+            </label>
+
+            <label for="SekValue" class="radio-button-container">
+              <p class="p-form-small">56 sek/month</p>
+              <input id="56SEK" type="radio" name="SekValue">
+              <span class="custom-radio"></span>
+            </label>
+          </div>
+        </div>
+
+        <p class="p-form">CONTACT INFORMATION</p>
+
+        <div class="person">
+          <label for="civicNr">
+            <p class="p-form-small">Civic nr </p>
+          </label>
+          <input 
+          id="civicNr" 
+          type="text" 
+          inputmode="numeric" 
+          placeholder="ÅÅÅÅMMDD-XXXX" 
+          name="civicNrValue" 
+          required>
+          *
+        </div>
+
+        <div class="person">
+          <label for="firstName">
+            <p class="p-form-small">First name</p>
+          </label>
+          <input id="firstName" type="text" name="firstNameValue" required>
```

Comment:
Like you did with the last input, it's a good idea to split the arguments to different lines if they are many:
```
<input
  id="firstName"
  type="text"
  name="firstNameValue"
  required>
```